@lowintelligence 6 tests failed but I only see the links to details for 3 tests. Let's try to fix that first.
- `Ubuntu Sanity`: Buildifier found a formatting error in tensorflow/core/kernels/BUILD. Please fix. The error message from the [log](https://source.cloud.google.com/results/invocations/3abeef23-0ea5-411f-ae97-51212dffe67d/targets/%2F%2Ftensorflow%2Ftools%2Fci_build:gen_ci_sanity_out/log):
```
=== Sanity check step 5 of 14: do_buildifier (buildifier check) ===
Running do_buildifier on 445 files
tensorflow/core/kernels/BUILD # reformat
buildifier took 1 s
FAIL: buildifier found errors and/or warnings in above BUILD files.
buildifier suggested the following changes:
1269c1269
<     ]
---
>     ],
Please fix manually or run buildifier <file> to auto-fix.
```
- `Windows Bazel GPU`: There's a linking error involving `absl`. Maybe you need to add an absl dependency in the BUILD file? Error message from the [log](https://source.cloud.google.com/results/invocations/c861f2d6-5e63-4ae0-8fad-49a5e7999045/log):
```
ERROR: T:/src/github/tensorflow/tensorflow/contrib/image/BUILD:115:1: Linking of rule '//tensorflow/contrib/image:python/ops/_distort_image_ops.so' failed (Exit 1120): link.exe failed: error executing command
...
lib_distort_image_ops_gpu.a(adjust_hsv_in_yiq_op_gpu.cu.o) : error LNK2019: unresolved external symbol "public: void __cdecl absl::Mutex::ReaderLock(void)" (?ReaderLock@Mutex@absl@@QEAAXXZ) referenced in function "public: __cdecl absl::ReaderMutexLock::ReaderMutexLock(class absl::Mutex *)" (??0ReaderMutexLock@absl@@QEAA@PEAVMutex@1@@Z)
lib_distort_image_ops_gpu.a(adjust_hsv_in_yiq_op_gpu.cu.o) : error LNK2019: unresolved external symbol "public: void __cdecl absl::Mutex::ReaderUnlock(void)" (?ReaderUnlock@Mutex@absl@@QEAAXXZ) referenced in function "public: __cdecl absl::ReaderMutexLock::~ReaderMutexLock(void)" (??1ReaderMutexLock@absl@@QEAA@XZ)
bazel-out/x64_windows-opt/bin/tensorflow/contrib/image/python/ops/_distort_image_ops.so : fatal error LNK1120: 2 unresolved externals
```
- `Ubuntu Python 3`: Got segfaults in keras backend test. It might just be flaky test and unrelated to this PR. Let's just see what happen after we rerun the tests first. Error message from [log](https://source.cloud.google.com/results/invocations/5788ff76-23e1-4d65-b972-25fc70b8a7da/log) for reference:
```
INFO: From Testing //tensorflow/python/keras:backend_test (shard 1 of 4):
==================== Test output for //tensorflow/python/keras:backend_test (shard 3 of 4):
external/bazel_tools/tools/test/test-setup.sh: line 310:    18 Segmentation fault      (core dumped) "${TEST_PATH}" "$@" 2>&1
================================================================================
==================== Test output for //tensorflow/python/keras:backend_test (shard 1 of 4):
external/bazel_tools/tools/test/test-setup.sh: line 310:    19 Segmentation fault      (core dumped) "${TEST_PATH}" "$@" 2>&1
================================================================================
==================== Test output for //tensorflow/python/keras:backend_test (shard 1 of 4):
external/bazel_tools/tools/test/test-setup.sh: line 310:    19 Segmentation fault      (core dumped) "${TEST_PATH}" "$@" 2>&1
================================================================================
==================== Test output for //tensorflow/python/keras:backend_test (shard 1 of 4):
external/bazel_tools/tools/test/test-setup.sh: line 310:    19 Segmentation fault      (core dumped) "${TEST_PATH}" "$@" 2>&1
================================================================================
```

I'm on vacation today and won't have access to computers all day, but I can check this again tonight. In the meanwhile, please feel free to ask @gbaned to help rerun the tests.

--- a/tensorflow/core/kernels/BUILD
+++ b/tensorflow/core/kernels/BUILD
@@ -1266,7 +1266,7 @@ tf_kernel_library(
     prefix = "unique_op",
     deps = ARRAY_DEPS + [
         "@com_google_absl//absl/container:flat_hash_map",
-    ]
+    ],
 )
 
 tf_kernel_library(

